app/views/paye/paye_your_income_and_taxes.scala.html

```diff
@@ -0,0 +1,83 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.ApplicationConfig
+@import view_models.paye.PayeYourIncomeAndTaxes
+@import layouts.{govuk_template => hmrcGovUkTemplate}
+@import uk.gov.hmrc.play.partials.FormPartialRetriever
+@import uk.gov.hmrc.play.views.html.{layouts => uiLayouts}
+@import controllers.auth.PayeAuthenticatedRequest
+
+@(viewModel:PayeYourIncomeAndTaxes)(implicit lang: Lang, messages: Messages, request: PayeAuthenticatedRequest[_], formPartialRetriever: FormPartialRetriever)
+
+@headerContent = {
+
+}
+
+@includes.taxs_main(title = Messages("paye.ats.summary.title") + Messages("generic.to_from", viewModel.taxYearStart, viewModel.taxYearEnd), "", contentHeader = Option(headerContent), mainClasses = Some("selectYear")) {
+
+@helpers.back_link(Messages("generic.back"))
+
+	<h1 class="heading-xlarge">
+			@Html(Messages("paye.ats.summary.title"))
+	    <span class="block-heading">
+	      @Html(Messages("paye.generic.tax_year_heading", viewModel.taxYear.toString, (viewModel.taxYear + 1).toString))
+	    </span>
+    </h1>
+
+    <p id="intro-paragraph">@Messages("paye.ats.summary.lede")</p>
+
+    <section id="taxable-income">
+        <h2>@Html(Messages("paye.ats.summary.total_income.title"))</h2>
+        <p>
+            <a href="#">
+                <span class="bold-xlarge" id="income-before-tax-amount">@includes.currency_amount(viewModel.incomeBeforeTax, true)</span><br>
+                <span class="bold-small" aria-hidden="true">@Html(Messages("paye.ats.summary.total_income.link"))</span>
```

Comment:
This looks odd, is it a mega link over two lines? I'm not sure if this passes GDS standards, I think a screen reader might get confused.

Reply:
@russellthorn Can you please address this?

Reply:
This sadly, is being implemented as a copy of SA version. Once again changing this isn't in scope in what we're currently calling the MVP. I don't like it and there are better things to come in the prototype (which I'm not sure have been tested yet). I am constantly trying to make changes and improvements, but as it stands with the current crazy work flow, changes are being frowned upon and I limited on my influence and quite honestly, the politics is getting tiring. We're trying to ship a copy of SA, that I don't think is fit for purpose.

Reply:
To answer you screen reader concern, this reads ok with VoiceOver and as I say it's a copy of what's already live in SA.